A/Aeron/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Aeron"
+version = v"1.40.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/real-logic/aeron.git", "1cda80dbcd346ee0409fec5328956292614460df"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+apk update
+apk add openjdk11 hdrhistogram-c-dev
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DAERON_ENABLE_NONSTANDARD_OPTIMIZATIONS=OFF AERON_TESTS=OFF aeron
+make
```

Comment:
This will make the build less horribly slow
```suggestion
make -j${nproc}
```

But `make install` disappeared.